scylladb/sphinx-scylladb-theme
# Highlight current project

This pull request:

1. Highlights the current project on the sidebar:

![image](https://user-images.githubusercontent.com/9107969/218787413-dfa6e87a-8247-4ab5-8b62-9c60b9f0c417.png)

2. Adds "ScyllaDB Docs" in front of the breadcrumb list:

![image](https://user-images.githubusercontent.com/9107969/218787740-2f12b7b2-2d52-46ea-b52a-a2ced8dba408.png)

## How to test this PR

1. Clone this PR and go to the `docs` folder.
2. Run `make preview` or `make multiversionpreview`.
3. Open http://127.0.0.1:5500/
4. Check the sidebar and breadcrumbs.
